### ISSUE TITLE
2.1.2 No Keyboard Trap (Loaded Data)

### ISSUE THREAD

**Issue by AlanaBayless** [UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan#28]
**Were violation(s) found?**
(If Yes, describe in the comments.)
- [ ] No violations found
- [ ] Yes, violations found and all fixed
- [ ] Yes, violations found and some fixed while others are unresolved _(Add **6.Unresolved 508** label)_
- [ ] Yes, violations found but remain unresolved _(Add **6.Unresolved 508** label)_
- [ ] Flag as a potential problem _(Add **4.Help** label)_

---
### 2.1.2 No Keyboard Trap - Level A
If keyboard focus can be moved to a component of the page using a keyboard interface, then focus can be moved away from that component using only a keyboard interface, and, if it requires more than unmodified arrow or tab keys or other standard exit methods, the user is advised of the method for moving focus away. (Level A)

_Note:_ Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the Web page (whether it is used to meet other success criteria or not) must meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG20/#cc5).

[Understanding 2.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html)
[How to Meet 2.1.2](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131%2C133%2C211#no-keyboard-trap)
[How to Test 2.1.2](https://usfs.box.com/s/knn5w8jul8g0xicrnpa4cxzk07t8rc1d)

**Trusted Tester ID:**
- [x] Test 4.C no-keyboard-trap

**Accessibility Principle:** Operable (Keyboard Accessible)

**Comment by AlanaBayless** [comment]
The Loaded Data page pass 2.1.2 No Keyboard trap. There is no keyboard trap.

**Trusted Tester ID:**

- [X]  Test 4.C no-keyboard-trap

**Comment by AlanaBayless** [comment]
This page is QA Certified. There are no keyboard traps.
